### ISSUE TITLE
Give "program details" and "external site" links on homepage greater contrast

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1472]
These light blue links on a white background have a contrast ratio of just 2.5:1, when they should have a ratio of 4.5:1. Pick a new color for these links [and test it here](https://webaim.org/resources/contrastchecker/) to make sure the contrast is high enough.

![image](https://user-images.githubusercontent.com/77128518/123181907-1ed36580-d443-11eb-9749-0c536f0ec560.png)

## Original tester's report

> ACTUAL RESULT: 'Program details' and 'External site' links have insufficient color contrast of 2.5:1
> EXPECTED RESULT: Color contrast ratio for regular size text should be at least 4.5:1.
